Magento_Swatches/web/js/swatch-renderer.js

```diff
@@ -1021,8 +1021,13 @@ define([
                   .addClass($widget.options.classes.loader);
           } else {
               //Category View
-              $this.parents('.product-item-info').find('.product-image-photo')
-                  .addClass($widget.options.classes.loader);
+              $this.parents('.product-item-info')
+                  .find('.lazyload-wrapper')
+                  .append(`
+                    <div class="loader loader--visible">
+                      <div class="loader__circle"></div>
+                    </div>
+                  `);
```

Comment:
IMO you don't need literal string here, you can use regular quotes

Reply:
I need it for multiline strings, it's more readable